### DISCUSSION TITLE
Small variant for select component

### DISCUSSION THREAD

**Discussion by GraceFinlay** [Telefonica/mistica-design#1021] [Proposals]
Buenas tardes,

Mi nombre es Grace y estoy trabajando en el proyecto de ecommerce global y os voy a indicar una serie de dudas relacionadas con Mística que han surgido en el desarrollo del proyecto.

# Contexto

El proyecto que está en marcha quiere incorporar unas mejoras en la tienda para que diferentes usuarios puedan comprar diferentes dispositivos. Estas mejoras que se quieren introducir son entre otras la mejora de los flujos de usuario y diferentes funcionalidades.

# Problema

Para este proyecto se requiere utilizar Mística y observamos que tenemos unas necesidades concretas en cuanto a componentes. Para nosotros es importante obtener feedback de vuestra parte con el fin de entender cómo podríamos implementar el proyecto usando Mística y qué cubra las necesidades planteadas.En este caso queremos diseñar la card que incluya el producto que el usuario ha añadido a la cesta.

# Propuesta
![Producto cesta](https://user-images.githubusercontent.com/99174134/214356487-b8a869bc-e8de-4f5d-9545-d4a38b776948.png)

Para diseñar este componente entendemos que debemos partir de átomos pero no logro encontrar unos dropdown tan pequeños.

Estamos abiertos a cualquier feedback que nos podáis dar para abordar el proyecto.

De antemano muchas gracias.

**Comment by yceballost** [comment]
Buenas tardes, Grace!

No nos hemos olvidado de ti jaja hemos estado pensando sobre esto y también viéndolo con el equipo de desarrollo para alinearnos y creemos que debe existir una versión "small" para los select.

Lo añadimos al backlog

Aquí se abren varios caminos para avanzar con esto:
1. Lo añadimos a nuestro backlog y lo cogemos en cuanto tengamos hueco
2. Trabajamos de la mano para que evolucionéis vosotros las specs de diseño y luego lo desarrolla el equipo de desarrollo de Mística
3. Trabajamos de la mano para que evolucionéis vosotros las specs de diseño y vuestro equipo de desarrollo hace una evolución en el código
